index.org

```diff
@@ -1788,7 +1788,75 @@ Modding documentation for Mindustry.
    * ~units~ Blocks that create units;
    * ~upgrade~ Things that upgrade the player such as mech pads;
    * ~effect~ Things for storage or passive effects.
-** TODO Zone
+** Zone
+    You can add your mod's campaign map by this
+    
+    | field              | type            | default    |
+    |--------------------+-----------------+------------|
+    | generator          | [[Generator]]   |            |
+    | baseLaunchCost     | [[ItemStack]]   |            |
+    | launchCost         | [[ItemStack]]   |            |
+    | startingItems      | [[ItemStack]]   |            |
+    | conditionWave      | int             | MAX_VALVE  |
+    | alwaysUnlocked     | boolean         |            |
+    | launchPeriod       | int             | 10         |
+    | loadout            | [[Loadout]]     | basicShard |
+    | resources          | [[Item]]        |            |
+    | rules              | [[Rules]]       |            |
+    | requirements       | [[Objective]]   |            |
+    | configureObjective | [[Objective]]   |            |
+    | defaultStartingItems | [[ItemStack]] |            |
+    
+*** Loadout
+    Loadout is the blueprint when your core landed.
+    
+    In Loadout:
+    | '>' is stands for conveyor face to right |
+    | '^' is stands for conveyor face to up |
+    | '<' is stands for conveyor face to left |
+    | 'v' is stands for conveyor face to down |
+    | '1' is stands for coreShard |
+    | '2' is stands for coreFoundation |
+    | '3' is stands for coreNucleus |
+    | 'C' is stands for mechicalDrill with four copper ore |
+    
+    Note:
+    You must set a core in your custom Loadout
+    
+    Built-in loadout:
+    * ~basicShard~
+        ("  ###  ",
+        "  #1#  ",
+        "  ###  ",
+        "  ^ ^  ",
+        " ## ## ",
+        " C# C# ")
```

Comment:
This should be inside a code block `#+BEGIN_SRC fundamental` and `#+END_SRC` to maintain mono font spacing. I don't understand what it's supposed to be either, `(` isn't used at all in the context of JSON.